Worker/scanJob.py

```diff
@@ -0,0 +1,38 @@
+from pymongo import MongoClient
+import clamd
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+resultsPath = os.getenv("RESULTS_PATH")
+cd = clamd.ClamdUnixSocket()
+
+mongodbHost = os.getenv("MONGODB_HOST")
+mongodbPort = int(os.getenv("MONGODB_PORT"))
+
+client = MongoClient(mongodbHost, mongodbPort)
+scan8 = client['scan8']
+queuedScans = scan8['queuedScans']
+runningScans = scan8['runningScans']
+completedScans = scan8['completedScans']
+
+# RQ job
+def scan(filePath):
+    id = filePath.split("/")[-2]
+    queued = list(queuedScans.find({"_id": id}))
+    if(len(queued) != 0):
+        runningScans.insert_one(queued[0])
+        queuedScans.delete_one({"_id": id})
+    result = cd.scan(filePath)
+    
+    print(result)
```

Comment:
@maanas-talwar Do we need to print the results ?

Reply:
@1209973 I'm printing the results to the console for now. I will be working on saving the results to a file next.

Reply:
Sounds good?

Reply:
yeap